service/frontend/workflowHandler.go

```diff
@@ -883,7 +884,9 @@ func (wh *WorkflowHandler) PollWorkflowTaskQueue(ctx context.Context, request *w
 // event in the history for that session.  Use the 'taskToken' provided as response of PollWorkflowTaskQueue API call
 // for completing the WorkflowTask.
 // The response could contain a new workflow task if there is one or if the request asking for one.
-func (wh *WorkflowHandler) RespondWorkflowTaskCompleted(ctx context.Context, request *workflowservice.RespondWorkflowTaskCompletedRequest) (_ *workflowservice.RespondWorkflowTaskCompletedResponse, retError error) {
+func (wh *WorkflowHandler) RespondWorkflowTaskCompleted(ctx context.Context,
+	request *workflowservice.RespondWorkflowTaskCompletedRequest) (_ *workflowservice.RespondWorkflowTaskCompletedResponse,
+	retError error) {
```

Comment:
```go
func (wh *WorkflowHandler) RespondWorkflowTaskCompleted(
	ctx context.Context,
	request *workflowservice.RespondWorkflowTaskCompletedRequest)
(_ *workflowservice.RespondWorkflowTaskCompletedResponse, retError error) {
```

Reply:
I did want to start this, but yes, let's make some standart here. Seems like nor `gofmt` neither `goland` are not care much about it. So there are two cases:
1. One line. All parameters are on the saneline, all return values are on the same line, no empty line before func body:
```go
func convertString(input string) string {
    return strings.ToUpper(input)
}
```
2. Many lines. Method/func name and all parameters are on separate line, each has comma at the end and there is a blank line before body. Return values are still on the same line though as we almost don't have more than 2:
```go
func (wh *WorkflowHandler) RespondWorkflowTaskCompleted(
    ctx context.Context,
    request *workflowservice.RespondWorkflowTaskCompletedRequest,
) (_ *workflowservice.RespondWorkflowTaskCompletedResponse, retError error) {

    defer ...
```